Doc/library/asyncio-platforms.rst

```diff
@@ -23,6 +23,8 @@ All Platforms
 Windows
 =======
 
+The default event loop on Windows is :class:`ProactorEventLoop`.
```

Comment:
Just put one `.. versionchanged:: 3.8` here, there's no point in covering this info twice in this section.

Reply:
done